### PR TITLE
[noref] Add section parameter to retrieve items example

### PR DESCRIPTION
**Pull request by shagabutdinov**
Bugfix for this report: https://constructor.slack.com/archives/C1HVAUJ0N/p1548644762054100

### CODE REVIEW

**Comment by julioolvr** [comment]
Seems like we're missing the `section` query param for retrieving and removing searchandizing rules as well, mind adding it while you're at it?

File: source/includes/_searchandizing.add_rule.md [hunk 1]
```diff
@@ -54,4 +54,5 @@ id | The id for the item you'd like added to the position and query in question.
 
 Parameter | Required? | Description
 --------- | ----------- | ----------
+section | Yes | The section   Your autocomplete suggestions can have multiple sections like “Products” and “Search Suggestions”. This indicates which section this searchandizing rule is for. See your dashboard for the section names to use.
```
Comment (julioolvr) [inline source/includes/_searchandizing.add_rule.md]: There is another parameter with this description where "your dashboard" is a link to the dashboard. Should we make this one a link too?

Comment (shagabutdinov) [reply source/includes/_searchandizing.add_rule.md]: Thanks, did it!

**Comment by shagabutdinov** [comment]
The PR is ready to be reviewed again.

File: source/includes/_searchandizing.add_rule.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ id | The id for the item you'd like added to the position and query in question.
 
 ### JSON Parameters
 
-Parameter | Required? | Description
+Parameter | Required?   | Description
 --------- | ----------- | ----------
+section   | Yes         | Your autocomplete suggestions can have multiple sections like “Products” and “Search Suggestions”. This indicates which section this searchandizing rule is for. See [your dashboard](http://app.constructor.io/dashboard) for the section names to use.
```
Comment (julioolvr) [inline source/includes/_searchandizing.add_rule.md]: nitpick: we can use `https` instead of relying on the redirect, right?

**Comment by shagabutdinov** [comment]
Sure, thanks! Did it.

--
Leonid Shagabutdinov,
+7 905 937 6988

**Pull request merged**